pkg/universe.dagger.io/x/contact@kjuulh.io/rust/container.cue

```diff
@@ -0,0 +1,29 @@
+package rust
+
+import (
+	"dagger.io/dagger"
+	"universe.dagger.io/docker"
+)
+
+#Container: {
+	// Container name
+	name: *"rust_publisher" | string
```

Comment:
I've removed it. It was added because both go and dotnet had done it, but as it is only really used for calculating caches in the other actions, I will leave it out for now, and add it in a future pr, when needed

Reply:
The name was used to avoid cache issue with `#CacheDir`.
[See how name is used](https://github.com/dagger/dagger/blob/main/pkg/universe.dagger.io/go/container.cue#L35)